docs/environment.yml

```diff
@@ -1,10 +1,11 @@
 name: qtconsole_docs
 channels:
+  - astrofrog
```

Comment:
Oh, they're there now. Maybe I had an old copy cached or something. I was seeing only the `Session` options in the doc - everything else was completely missing. I'm not really sure how it's working without pyqt installed, though :confused: .